CRM/Civicase/APIHelpers/CustomGroups.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Custom Groups API Helper Class.
+ */
+class CRM_Civicase_APIHelpers_CustomGroups {
+
+  /**
+   * Returns a list of all active custom groups.
+   *
+   * @return array
+   *   Custom Group Api response.
+   */
+  public static function getAllActiveGroups() {
+    return civicrm_api3('CustomGroup', 'get', [
+      'extends' => $params['entity_type'],
+      'options' => $params['options'],
+      'is_active' => 1,
+    ]);
+  }
+
+  /**
+   * Returns the custom group ID for the given custom group name.
+   *
+   * Does not support multiple custom group names. Returns NULL if no group
+   * was found.
+   *
+   * @return int|null
+   *   A custom group id or NULL.
+   */
+  public static function getGroupIdFromSingleGroupName($params) {
```

Comment:
Can we modify the parameter this accepts to be just `$customGroupName`. There is no need to pass an array and if we do this, this function is easily reusable elsewhere without having to pass an array with some predefined parameter value.